src/onnxruntime_loader.cc

```diff
@@ -191,8 +192,13 @@ OnnxLoader::LoadSession(
           loader->env_, ort_style_model_str.c_str(), model.size(),
           session_options, session);
     } else {
-      status = ort_api->CreateSession(
-          loader->env_, ort_style_model_str.c_str(), session_options, session);
+      std::ifstream bytes_stream(ort_style_model_str.c_str(), std::ios::in | std::ios::binary | std::ios::ate);
```

Comment:
Can you add a comment mentioning that the reason to pass the ONNX file as a binary is to avoid the TRT cache from being regenerated?